### ISSUE TITLE
{Linked:Bug11845587}[Adaptive Cards > Designer] - NVDA is announcing grouping label name for all the check boxes.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8829]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 24H2 (OS Build 26052.1000)
Browser Version: Version 123.0.2400.1 (Official build) dev (64-bit)
Screen Reader: NVDA
Tool: Accessibility Insight For Web

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Run the tool and switch the toggle On for the heading and observe the issue.

**Actual Result:**
NVDA is announcing grouping label name for all the check boxes.
NVDA Speech: 
Add Causes If cause not listed, please provide more details in the previous page. Absorption check box not checked.
Add Causes If cause not listed, please provide more details in the previous page Bodily Reaction check box not checked.
Add Causes If cause not listed, please provide more details in the previous page Caught in Under or Between check box not checked.

**Expected Result:**
NVDA should not announce grouping label name for all the check boxes.
NVDA should announce like Add Causes If cause not listed please provide more details in the previous page Absorption check box not checked, Bodily Reaction check box not checked, Caught in Under or Between check box not checked.

**User Impact:**
Users who rely on Screen Readers will get impacted if NVDA is announcing grouping label name for all the check boxes as they will get irretate.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/179b073f-bdfe-49c2-95ff-32845b495d8f

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/280ce58b-b17d-4fcf-b817-e6926c4ae3eb)

### Card JSON

```JSON
{
    "schema": "http=//adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.5",
    "body": [
        {
            "id": "",
            "isVisible": false,
            "separator": false,
            "spacing": "default",
            "type": "Container",
            "bleed": false,
            "items": [
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "width": "32px",
                    "type": "Image",
                    "height": "32px",
                    "horizontalAlignment": "center",
                    "size": "auto",
                    "style": "default",
                    "url": "https://localhost:4321/node_modules/myhub-viva-library/dist/LoadingSpinner_231a701aee45eda2616de6679644dd24.gif",
                    "altText": "Loading indicator"
                },
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "TextBlock",
                    "style": "default",
                    "wrap": false,
                    "text": "Loading... Please wait.",
                    "horizontalAlignment": "center",
                    "size": "default"
                }
            ],
            "verticalContentAlignment": "center",
            "rtl": false,
            "sizes": {
                "imageSize": "32px",
                "fontSize": "default"
            }
        },
        {
            "id": "",
            "isVisible": true,
            "separator": false,
            "spacing": "default",
            "type": "Container",
            "bleed": false,
            "items": [
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "Container",
                    "bleed": false,
                    "items": [
                        {
                            "id": "",
                            "isVisible": true,
                            "separator": false,
                            "spacing": "none",
                            "width": "stretch",
                            "type": "Container",
                            "bleed": true,
                            "items": [],
                            "verticalContentAlignment": "center",
                            "rtl": false,
                            "backgroundImage": "https://localhost:4321/node_modules/myhub-viva-library/dist/HeaderLine_77d65ccd075fa6644bc29b64bd426343.png"
                        },
                        {
                            "id": "",
                            "isVisible": true,
                            "separator": false,
                            "spacing": "none",
                            "type": "ColumnSet",
                            "bleed": false,
                            "columns": [
                                {
                                    "type": "Column",
                                    "separator": false,
                                    "spacing": "none",
                                    "verticalContentAlignment": "center",
                                    "width": "stretch",
                                    "isVisible": true,
                                    "rtl": false,
                                    "items": [
                                        {
                                            "id": "",
                                            "isVisible": true,
                                            "separator": false,
                                            "spacing": "none",
                                            "type": "TextBlock",
                                            "style": "heading",
                                            "wrap": true,
                                            "text": "Add Causes"
                                        }
                                    ]
                                }
                            ]
                        }
                    ],
                    "verticalContentAlignment": "center",
                    "rtl": false,
                    "headerType": "basic"
                },
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "default",
                    "type": "Container",
                    "bleed": false,
                    "items": [
                        {
                            "id": "impactViewRecordId",
                            "isVisible": true,
                            "separator": false,
                            "spacing": "Medium",
                            "width": "stretch",
                            "type": "Input.ChoiceSet",
                            "isRequired": false,
                            "isMultiSelect": true,
                            "style": "expanded",
                            "wrap": true,
                            "choices": [
                                {
                                    "title": "Absorption",
                                    "value": "ABS"
                                },
                                {
                                    "title": "Bodily Reaction",
                                    "value": "BR"
                                },
                                {
                                    "title": "Caught In, Under, or Between",
                                    "value": "CI"
                                },
                                {
                                    "title": "Driving Accident",
                                    "value": "DA"
                                },
                                {
                                    "title": "Exposure to, Contact with",
                                    "value": "ET"
                                },
                                {
                                    "title": "Ingestion",
                                    "value": "ING"
                                },
                                {
                                    "title": "Inhalation",
                                    "value": "INH"
                                },
                                {
                                    "title": "Overexertion / Ergonomic",
                                    "value": "OE"
                                },
                                {
                                    "title": "Rubbed or Abraded",
                                    "value": "ROA"
                                },
                                {
                                    "title": "Slip, Trip, Fall",
                                    "value": "STF"
                                },
                                {
                                    "title": "Struck Against / Struck By",
                                    "value": "SASB"
                                }
                            ],
                            "value": "",
                            "label": "Add Causes, If cause not listed, please provide more details in the previous page."
                        }
                    ],
                    "verticalContentAlignment": "center",
                    "rtl": false
                },
                {
                    "id": "",
                    "isVisible": true,
                    "separator": false,
                    "spacing": "large",
                    "type": "ActionSet",
                    "actions": [
                        {
                            "style": "positive",
                            "mode": "primary",
                            "isEnabled": true,
                            "id": "impactCausesSelectButtonId",
                            "title": "Select Cause(s)",
                            "tooltip": "Click here to select the causes and navigate to reporting step",
                            "type": "Action.Submit",
                            "associatedInputs": "auto",
                            "data": {
                                "id": "impactCausesSelectButtonId"
                            }
                        }
                    ]
                }
            ],
            "verticalContentAlignment": "center",
            "rtl": false
        }
    ],
    "actions": [],
    "refresh": null
}
```

**Comment by vagpt** [comment]
Originally tracking with below URL - 

https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11845587

**Comment by anna-dingler** [comment]
Hi @vagpt! I made a change in the middle of March that impacts the checkbox behavior. Can you confirm that this bug still occurs as I do not see this happening with Narrator?

**Comment by anna-dingler** [comment]
@vagpt Any updates?

**Comment by vagpt** [comment]
Hi @anna-dingler,

Sorry for the delayed response.

We have checked it again with NVDA and Narrator screen reader and getting the same issue at our end. Attaching the screenshot for your reference.

**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/7e6a07cd-547e-4bfe-91ef-4c99b1721797)

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/606cd182-a88b-45a4-8fce-cd1983270133)
